### PR TITLE
Match branding some more

### PR DESCRIPTION
**Pull request by bogotesr**
App name is now SideStore instead of AltStore and navigation bar at the bottom is purple instead of blue for accent color

### CODE REVIEW

**Comment by Spidy123222** [comment]
I have added a fix for actions to work and I also updated the branch to be up to latest.

**Comment by Spidy123222** [comment]
> App name is now SideStore instead of AltStore and navigation bar at the bottom is purple instead of blue for accent color

@bogotesr Is there a reason this still is a draft is there still stuff you need to do. It be nice to have a list of stuff you need to change so contributors can also help out.

**Comment by N-Games** [comment]
I've just tested this build, and all the commits are successfully integrated in the app

**Comment by jkcoxson** [comment]
The merge conflicts are easy to fix, but this is still marked as a draft. Is that true?

**Comment by Spidy123222** [comment]
> The merge conflicts are easy to fix, but this is still marked as a draft. Is that true?

It's technically ready. But just recently got bogoster that it's fine to merge now.

**Review by SternXD — COMMENTED** [review]
It seems good to me but the conflict is a problem once that gets resolved I'll give my approval

**Comment by jkcoxson** [comment]
All that needs to be done for the conflict is to delete the file. My PR removed all the server code, and that’s one of them.

File: AltStore/Authentication/Authentication.storyboard [hunk 1]
```diff
@@ -380,7 +380,7 @@
...
                                                         <nil key="highlightedColor"/>
                                                     </label>
-                                                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Apps are refreshed in the background when on same Wi-Fi as AltServer." textAlignment="natural" lineBreakMode="tailTruncation" numberOfLines="2" baselineAdjustment="alignBaselines" minimumScaleFactor="0.5" translatesAutoresizingMaskIntoConstraints="NO" id="HU5-Hv-E3d">
+                                                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Apps are refreshed in the background While your on SideStore VPN!" textAlignment="natural" lineBreakMode="tailTruncation" numberOfLines="2" baselineAdjustment="alignBaselines" minimumScaleFactor="0.5" translatesAutoresizingMaskIntoConstraints="NO" id="HU5-Hv-E3d">
```
Comment (jkcoxson) [inline AltStore/Authentication/Authentication.storyboard:383]: This shouldn’t be capitalized

File: AltStore/Info.plist [hunk 2]
```diff
@@ -90,7 +90,7 @@
 		<string>_altserver._tcp</string>
 	</array>
 	<key>NSLocalNetworkUsageDescription</key>
-	<string>SideStore uses the local network to find and communicate with SideServer.</string>
+	<string>SideStore uses the local network to find and communicate with your device.</string>
```
Comment (jkcoxson) [inline AltStore/Info.plist:93]: Pretty sure this whole entitlement can just be removed

File: AltStore/Server/Server.swift [hunk 3]
```diff
@@ -17,9 +17,9 @@ enum ConnectionError: LocalizedError
     var failureReason: String? {
         switch self
         {
-        case .serverNotFound: return NSLocalizedString("Could not find AltServer.", comment: "")
```
Comment (jkcoxson) [inline AltStore/Server/Server.swift:20]: Delete this entire file

File: app.json [hunk 4]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "SideStore Offical",
-  "identifier": "com.SideStore.AltStore",
```
Comment (jkcoxson) [inline app.json:3]: Have you tested this that it doesn’t cause issues with installing and refreshing?

**Review by jkcoxson — CHANGES_REQUESTED** [review]
Please address the comments

File: app.json (hunk 4 above)
Comment (SternXD) [reply app.json:3]: I'll test this again

Comment (Spidy123222) [reply app.json:3]: This wouldn't cause issues with installing or refreshing as this is exclusively only for the update system and sidestore having a store app.

Comment (SternXD) [reply app.json:3]: https://github.com/SideStore/SideStore/actions/runs/3381406451 using this i confirmed it works

File: AltStore/Authentication/Authentication.storyboard (hunk 1 above)
Comment (Spidy123222) [reply AltStore/Authentication/Authentication.storyboard]: The latest Commit should resolve this issue.

File: AltStore/Info.plist (hunk 2 above)
Comment (Spidy123222) [reply AltStore/Info.plist]: The latest Commit should resolve this issue.

File: AltStore/Server/Server.swift (hunk 3 above)
Comment (Spidy123222) [reply AltStore/Server/Server.swift:20]: The latest Commit should resolve this issue.

**Comment by jkcoxson** [comment]
If this builds, lgtm

**Review by jkcoxson — APPROVED** [review]

**Review by SternXD — APPROVED** [review]

**Pull request merged**